### PR TITLE
Add notes for Pittsburgh CG meeting

### PR DESCRIPTION
**Pull request by tlively**

### CODE REVIEW

File: main/2024/CG-06.md
Comment (dschuff) [inline main/2024/CG-06.md:77]: I think these agenda diffs are reverts of the agenda changes made after this file landed in the repo but before the meeting.

Comment (tlively) [reply main/2024/CG-06.md]: Thanks, fixed. I left off the link to Luke's slides because it appears again alongside the notes for that session.

File: main/2024/CG-06.md
Comment (dschuff) [inline main/2024/CG-06.md:273]:
```suggestion
Andreas Rossberg Presenting Custom annotations [slides](/2024-06-05-rossberg-annotations.pdf)
```

File: main/2024/CG-06.md
Comment (dschuff) [inline main/2024/CG-06.md:343]: @yuri91 do you have a slides link?

File: main/2024/CG-06.md
Comment (dschuff) [inline main/2024/CG-06.md:722]: @eqrion do you have a slides link?

File: main/2024/CG-06.md
Comment (dschuff) [inline main/2024/CG-06.md:878]: @dtig @bvisness do you have a slide link?

File: main/2024/CG-06.md
Comment (dschuff) [inline main/2024/CG-06.md:1203]:
```suggestion
AR Presenting [slides](presentations//2024-06-06-rossberg-spectec.pdf)
```

File: main/2024/CG-06.md
Comment (dschuff) [inline main/2024/CG-06.md:1302]: @frank-emrich do you have a link to the slides?

File: main/2024/CG-06.md
Comment (dschuff) [inline main/2024/CG-06.md:1566]: @camio do you want to publish your slides?

File: main/2024/CG-06.md
Comment (dschuff) [inline main/2024/CG-06.md:1789]: @sbc100 do you have a link to your slides?

File: main/2024/CG-06.md
Comment (yuri91) [reply main/2024/CG-06.md:343]: https://drive.google.com/file/d/12ojxNtRrugs-4MrLqOghCCn2fJbfmqcl/view?usp=sharing

File: main/2024/CG-06.md
Comment (bvisness) [reply main/2024/CG-06.md:878]: Here: https://docs.google.com/presentation/d/1n89xOMpZMpic9MBIuR3mkxABQCsYG6U7lr33TdWlVpQ/edit?usp=sharing

File: main/2024/CG-06.md
Comment (tlively) [inline main/2024/CG-06.md:343]:
```suggestion
YI presenting branch hinting [slides](https://drive.google.com/file/d/12ojxNtRrugs-4MrLqOghCCn2fJbfmqcl/view)
```

File: main/2024/CG-06.md
Comment (tlively) [inline main/2024/CG-06.md:878]:
```suggestion
DG & BV presenting [slides](https://docs.google.com/presentation/d/1n89xOMpZMpic9MBIuR3mkxABQCsYG6U7lr33TdWlVpQ/edit?usp=sharing); notes not already in slides:
```

File: main/2024/CG-06.md
Comment (tlively) [inline main/2024/CG-06.md:1302]:
```suggestion
FE presenting [slides](presentations/2024-06-06-emrich-wasmfx-stack-switching.pdf)
```

File: main/2024/CG-06.md
Comment (tlively) [inline main/2024/CG-06.md:1566]:
```suggestion
DVS presenting [slides](presentations/2024-06-06-sankel-big-wasm-binaries.pdf). (I am referring to him as DVS to be distinct from Derek.)
```

File: main/2024/CG-06.md
Comment (tlively) [inline main/2024/CG-06.md:273]:
```suggestion
Andreas Rossberg Presenting Custom annotations [slides](presentations/2024-06-05-rossberg-annotations.pdf)
```

**Review by dschuff — APPROVED** [review]
This LGTM; if you want to wait for the rest of the slides, or land it now and add links in followup commits, either way is fine with me.

File: main/2024/CG-06.md
Comment (tlively) [reply main/2024/CG-06.md:1789]: Friendly bump @sbc100

File: main/2024/CG-06.md
Comment (tlively) [reply main/2024/CG-06.md:722]: Friendly bump @eqrion

Comment (eqrion) [reply main/2024/CG-06.md:722]: https://docs.google.com/presentation/d/111gkNGmzOtATU7daMKBjZ6mBqbIXF6jhd3e1542-0xo/edit#slide=id.g1ef40614c6f_0_21

Sorry, for the delay.

File: main/2024/CG-06.md
Comment (sbc100) [reply main/2024/CG-06.md:1789]: Sorry, I'm was sure I posted this already: https://docs.google.com/presentation/d/1LRIIOUHZoTm127fnM-BuUDjVLOgUUzvqISJgiI_fWeY/edit?usp=sharing

**Pull request merged**
